### PR TITLE
Replace old time api java.util.time by new time api java.time

### LINKED ISSUE #5: DateTimeUtils use obsolete time API
java.util.Date is Java's time API since JDK 1.1. Since JDK 1.8, it is encouraged to use new time API: java.time, because it's thread-safe and more concise

### PR DESCRIPTION
**Pull request by anhoangphuc**

### CODE REVIEW

**Comment by anhoangphuc** [comment]
this pull request may close #5

File: src/main/java/woocommerce/utils/DateTimeUtils.java [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Date;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
 
 public class DateTimeUtils {
-    private static final SimpleDateFormat ISO_8601 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm");
+    private static final DateTimeFormatter ISO_8601 = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm");
```
Comment (weirdobeardo48) [inline src/main/java/woocommerce/utils/DateTimeUtils.java]: Woud you please add second to this format as we have discussed before?
Thanks a lot

**Review by weirdobeardo48 — CHANGES_REQUESTED** [review]
A little change as I have commented would be great!

**Comment by weirdobeardo48** [comment]
Thanks for your contribution @anhoangphuc , let's move on!

**Pull request merged**